script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -295,47 +298,47 @@ The faint light of the sun dances across my eyelids.
 [sha:1d37832ed57fd3dd1dc1f08f3fcf3eee56852c52]{
 -- Page 13, Offset 29029.
 -- 柔らかな朝の訪れ。
--- TRANSLATION HERE
+It is the soft arrival of morning.
 }
 [sha:f40f02d6512312f3f4f1495b4be2b7403f3613c0]{
 -- Page 14, Offset 29030.
 -- 寝床から起き上がり、時刻と曜日を確認する。
--- TRANSLATION HERE
+I get up from my bed to see what time and day it is.
 }
 [sha:2113a2e293299e2ae06061ef7d42e130dc14a6bb]{
 -- Page 14, Offset 29031.
 -- 午前７時前、金曜日。
--- TRANSLATION HERE
+Seven in the morning, Friday.
 }
 [sha:398cfd207a8507fd47ff90a9d3e8a81d9996bdac]{
 -- Page 14, Offset 29032.
 -- 体の調子はいいとは言えない。
--- TRANSLATION HERE
+My body isn't in the greatest of shapes.
 }
 [sha:ae7613f83665a444fb310cc8d73c3d4caef2e49b]{
 -- Page 14, Offset 29033.
 -- 疲れが取れていないのだろう。体中が気だるく、熱っぽい。
--- TRANSLATION HERE
+I'm probably still just tired. My whole body feels lethargic and feverish.
 }
 [sha:0b80b273f6e7be6cf682a52faa4427e318da677f]{
 -- Page 15, Offset 29034.
 -- 「う…………」
--- TRANSLATION HERE
+"Ugh..."
 }
 [sha:ab4f66d9fd1cc08615ef1f671b0343587a85798d]{
 -- Page 15, Offset 29035.
 -- 深呼吸をするだけで吐きそうになった。
--- TRANSLATION HERE
+Even just taking deep breaths makes me want to vomit.
 }
 [sha:179015a5ed91167f97314ebc82c8c9428efccfa8]{
 -- Page 15, Offset 29036.
 -- いつもの貧血ではなく、空腹からくる不快感だ。
--- TRANSLATION HERE
+This is not my usual anemia, it's the discomforting feeling of hunger.
```

Comment:
"it's the discomfort of an empty stomach" 
or
 "it's the uncomfortable feeling of hunger" (softer and less favorable)